torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1339,6 +1341,71 @@ def call_function(
         )
 
 
+class StrictModeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+        callable = args[0]
+        graph_checkpoint, checkpoint = tx.output.graph, tx.copy_graphstate()
+        (ret_val, ret_treespec), ret_graph, ret_lifted_freevars, = speculate_subgraph(
+            tx,
+            args[0],
+            args[1].unpack_var_sequence(tx),
+            {},
+            "strict_mode",
+            source_target=self.value,
+            manually_set_subgraph_inputs=False,
+            should_flatten_outputs=True,
+        )
+
+
+        strict_mode_nn_modules = tx.copy_graphstate().output.nn_modules
```

Comment:
no need to copy_graphstate any more. tx.output.nn_modules is good.

Reply:
Done